File: 프로그래머스/unrated/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.cpp
#include <string>
#include <vector>
#include <iostream>

using namespace std;

int solution(string t, string p) {
    int answer = 0;
    int n = p.length();
    long long int_p = stol(p);
    for(int i=0; i<t.length(); i++) {
        string str = t.substr(i,n);
        if(str.length() < n) continue;
        long long k = stol(str);
        cout<<k<<" ";
        if(k <= int_p) answer++;
    }
    return answer;
}

File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.js
function solution(t, p) {
    var answer = 0;
    let size = p.length;
    for(let i=0; i<t.length; i++) {
        let sub = t.substring(i,i+size);
        if(sub.length < size) break;
        console.log(sub);
        if(Number(sub) <= Number(p)) answer++;
    }
    return answer;
}